vertx-pg-client/src/main/java/io/vertx/pgclient/impl/RowImpl.java

```diff
@@ -228,7 +228,7 @@ private Object getEnum(Class enumType, int pos) {
         }
       }
     }
-    throw new ClassCastException();
+    return null;
```

Comment:
this CCE is here for a reason, e.g the class type is incorrect, so it should return null only if vall is null otheriwse it must throw CCE. It seems this case is not tested and it should be tested as well

Reply:
At first glance, I think you want to throw exception if incorrect class type, but I checked other Row implementation and see all of them return null. So not sure  what do you want to. 
It's ok, let me add check null before return null

Reply:
Done. Anyway, do you want to add this check for other Row implementations? @vietj

Reply:
yes the impl should be consistent in similar methods I think

Reply:
yes, I updated code for other implementations before